packages/core/register/src/hook.js

```diff
@@ -58,8 +58,10 @@ export default function register(opts = DEFAULT_CLI_OPTS) {
         return output;
       }
     } catch (e) {
+      /* eslint-disable no-console */
       console.error('@parcel/register failed to process: ', filename);
```

Comment:
This should probably use `@parcel/logger`, but I don't have a way of manually verifying it.